fastlane/Fastfile

```diff
@@ -20,12 +20,12 @@ default_platform(:ios)
 platform :ios do
   desc "Run Tests and Output Code Coverage"
   lane :test do |options|
-    workspace = options[:workspaceFilename]
+    workspace = "OneLogin.xcworkspace"
```

Comment:
Why was this changed? this value is passed in from the github action so if its wrong there we need change it on the action not here